tasks/write-to-a-windows-event-log/src/main.rs

```diff
@@ -0,0 +1,44 @@
+#[cfg(target_os = "windows")]
+mod bindings {
+    ::windows::include_bindings!();
+}
+
+const TOKEN_READ: u32 = 0x00020008;
+const EVENTLOG_WARNING_TYPE: u16 = 2;
+
+#[cfg(target_os = "windows")]
+fn main() {
+    use bindings::windows::win32::security::*;
+    use bindings::windows::win32::system_services::*;
+
+    unsafe {
+        let ph = GetCurrentProcess();
+        let mut th: HANDLE = HANDLE(0);
+        OpenProcessToken(ph, TOKEN_READ, &mut th);
```

Comment:
Make sure to check errors for all of these calls. The `windows` crate provides some neat extensions for turning `BOOL`s into `Result`s so that you can use the question mark operator with them.